qfdmo/models/action.py

```diff
@@ -15,11 +15,10 @@
 
 class ActionDirection(CodeAsNaturalKeyModel):
     class Meta:
```

Comment:
Si on fait remonter les constraints sur le parent l'héritage devra aussi se faire au niveau de la class Meta : 
```suggestion
    class Meta(CodeAsNaturalKeyModel.Meta):
```